docs/node/CATEGORIES/08-account.md

```diff
@@ -31,7 +31,7 @@ interface account {
 }
 
 interface AccountPagination {
-  start?: number
```

Comment:
During **run-time**, in some scenarios, actually `string` is used. However, because the parameter was passed as `any`, Typescript is unable to check whether the variable **is not** a number during **compile-time**. 


This example shows why `any` is not recommended. Because it disables the compile-time checking